### PR TITLE
First draft of spark pipeline

### PR DESCRIPTION
**Pull request by almussel**
Draft of spark pipeline for @benedictpaten to review

### CODE REVIEW

File: adam-pipeline/spark-toil-script.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+def tachyon_workers(job, numWorkers, masterIP):
+
+    for i in range(len(numWorkers)):
+        job.addService(TachyonWorkerService(masterIP))
+
+    job.addChildJobFn(hello_world)
+
+def hello_world(job):
+
+    print "Hello World"
```
Comment (heuermh) [inline adam-pipeline/spark-toil-script.py]: Is this where something like ```spark-shell <<<'sc.parallelize(1 to 1000).count()'``` might go?

File: adam-pipeline/spark-toil-script.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python2.7
+# Audrey Musselman-Brown
```
Comment (hannes-ucsc) [inline adam-pipeline/spark-toil-script.py]: Our standard copyright header should go here. Most files in Toil carry it.

File: adam-pipeline/spark-toil-script.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+# SERVICE CLASSES
+
+class MasterService(Job.Service):
+
+    def start(self):
+        log.write("start master\n")
+        log.flush()
+        self.sparkContainerID = check_output(["docker", "run", "--net='host'", "-d", "-e",\
+                                              "\"SPARK_MASTER_IP=$(hostname -f)\"", \
+                                              "computationalgenomicslab/apache-spark-master:1.5.0"])[:-1]
```
Comment (heuermh) [inline adam-pipeline/spark-toil-script.py]: This could be bumped to version 1.5.2, see https://github.com/BD2KGenomics/cgl-docker-lib/pull/58

**Comment by almussel** [comment]
This pull request is out of date and was replaced with #72

**Pull request closed without merging**